specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/managedInstanceOperations.json

```diff
@@ -57,6 +57,52 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/managedInstances/{managedInstanceName}/operations/{operationId}": {
+      "get": {
+        "tags": [
+          "ManagedInstanceOperations"
+        ],
+        "description": "Gets a management operation on a managed instance.",
+        "operationId": "ManagedInstanceOperations_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedInstanceNameParameter"
+          },
+          {
+            "name": "operationId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the specified managed instance operation.",
+            "schema": {
+              "$ref": "#/definitions/ManagedInstanceOperation"
+            }
+          },
+          "default": {
```

Comment:
Please reference the schema with [standard model here. Example](https://github.com/Azure/azure-rest-api-specs/blob/6a03b4217d5af3c3296de5cc27ab16a51481d882/specification/hybridcompute/resource-manager/Microsoft.HybridCompute/stable/2019-12-12/HybridCompute.json#L82)

Reply:
This API was done following already existing API for SQL DB which doesn't have this field as well. For reference, take a look at:  https://github.com/Azure/azure-rest-api-specs/blob/master/specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/cancelOperations.json.
At the previous swagger review of this API (which was abandoned due to issues with branch) there was a ask for creating new API for getting single operation, which I did even though SQL DB has swagger without it. 
Since we choose to align with SQL DB we agreed not to change anything else.